src/cms/views/pages/page_view.py

```diff
@@ -134,6 +135,9 @@ def get(self, request, *args, **kwargs):
         else:
             siblings = page.parent.children.all()
         context = self.get_context_data(**kwargs)
+        page_link = f"https://integreat.app/{region.slug}/{language.slug}/"
```

Comment:
```suggestion
        page_link = f"{WEBAPP_URL}/{region.slug}/{language.slug}/"
```

https://github.com/Integreat/integreat-cms/blob/3482ec00b5ef9c6393da4d5e5d3b0a2878f9c26e/src/backend/settings.py#L54